aip/auth/4117.md

```diff
@@ -237,15 +237,17 @@ The auth libraries and applications **must** follow the steps below:
   error, trigger an error that the auth library needs to be updated to handle
   this type of credentials.
 - If **imdsv2_session_token_url** is available, then fetch session token
-  from **imdsv2_session_token_url**.
+  from **imdsv2_session_token_url**. Note: only perform this step if you
+  need to communicate with the metadata server to fetch the region and/or
+  the security credentials
 - Check the environment variables in the following order (`AWS_REGION` and
   then the `AWS_DEFAULT_REGION`) to determine the AWS region. If found, skip
   using the AWS metadata server to determine this value.
 - If the region environment variable is not provided, use the **region_url**
   to determine the current AWS region. The API returns the zone name, e.g.
   `us-east-1d`. The region should be determined by stripping the last
   character, e.g. `us-east-1`.
-- Check the environment variables `ACCESS_ACCESS_KEY_ID`,
+- Check the environment variables `AWS_ACCESS_ACCESS_KEY_ID`,
```

Comment:
Two `ACCESS`?  The documentation for the CI says `AWS_ACCESS_KEY_ID`:

https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html